doc/src/pair_lj_sphere.rst

```diff
@@ -0,0 +1,124 @@
+.. index:: pair_style lj/sphere
+.. index:: pair_style lj/sphere/omp
+
+pair_style lj/sphere command
+============================
+
+Accelerator Variant: *lj/sphere/omp*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style style args
+
+* style = *lj/sphere*
+* args = list of arguments for a particular style
+
+.. parsed-literal::
+
+     *lj/sphere* args = cutoff
+       cutoff = global cutoff for Lennard Jones interactions (distance units)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/sphere 2.5
+   pair_coeff * * 1.0
+   pair_coeff 1 1 1.1 2.8
+
+Description
+"""""""""""
+
+The *lj/sphere* style compute the standard 12/6 Lennard-Jones potential,
+given by
+
+.. math::
+
+   E = 4 \epsilon \left[ \left(\frac{\sigma_{ij}}{r}\right)^{12} -
+       \left(\frac{\sigma_{ij}}{r}\right)^6 \right]
+                       \qquad r < r_c
+
+:math:`r_c` is the cutoff.
+
+This is the same potential function as used by the :doc:`lj/cut
+<pair_lj>` pair style, but the :math:`\sigma_{ij}` parameter is not set
+as a per-type parameter via the :doc:`pair_coeff command <pair_coeff>`,
+but taken from the per-atom radius attribute of :doc:`atom_style sphere
```

Comment:
Please clarify if you mean radius or diameter is used in setting sigma.  I think you mean diameter?
E.g. if two particles with radius = 0.5 are interacting, then sigmaij = 1.0 = ave of two diameters?
I think from the user perspective per-atom radius is often an internal parameter and data files and the set command, e.g. for atom_style sphere, specify diameters.